apps/wing/src/commands/test.ts

```diff
@@ -49,14 +49,48 @@ async function testOne(entrypoint: string) {
     const status = result.pass ? chalk.green("pass") : chalk.red("fail");
 
     const pathWithPadding = chalk.whiteBright(result.path.padEnd(longestPath));
-    const error = result.error ? `\n${chalk.red(result.error)}` : "";
+    const details = new Array<string>();
 
-    const sep = chalk.gray(" | ");
-    const row = [status, basename(entrypoint), pathWithPadding + error].join(
-      sep
-    );
+    // add any log messages that were emitted during the test
+    for (const log of result.traces.filter(t => t.type == "log")) {
+      details.push(chalk.gray(log.data.message));
```

Comment:
Would it be too verbose to include more from the trace? Maybe atleast `log.sourcePath` could be added here.

```suggestion
      details.push(chalk.gray(`[${log.sourcePath}] ${log.data.message}`));
```

Reply:
Since at the moment, `sourcePath` is going to be the test inflight context, it will result in a weird duplication: 
![Screenshot 2023-02-19 at 18 12 54](https://user-images.githubusercontent.com/598796/219960291-ac9d0ece-3c96-4654-84f4-1501066237aa.png)


Let's start without it and we can add later ....